src/Maestro/Maestro.Web/Maestro.Web.csproj

```diff
@@ -45,6 +45,9 @@
     <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" />
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" />
     <PackageReference Include="Octokit" />
+    <PackageReference Include="Swashbuckle.AspNetCore" />
```

Comment:
does adding this mean we're not using `Microsoft.AspNetCore.ApiVersioning.Swashbuckle` from dnceng-shared anymore? If so we should remove it from the csproj

Reply:
Yes, unfortunately. This API versioning we use is actually already deprecated and the new libraries are named completely different but it would be hard to map it onto our date-based versions so it will have to stay around until we re-do it quite a bit.